install/kubernetes/helm/subcharts/kiali/templates/deployment.yaml

```diff
@@ -72,5 +72,7 @@ spec:
       - name: kiali-configuration
         configMap:
           name: kiali
+{{- if .Values.nodeaffinityTemplate  }}
```

Comment:
What is this flag? We don't have it.
cc @morvencao

Reply:
@ymesika I've updated the PR to define correctly a variable based on the nodeaffinity template defined on istio parent chart https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/templates/_affinity.tpl#L3

I've tested it and looks working fine, let me know if you need anything else!

Reply:
@ivanmp91 Can you explain how you are using this? I'm interested in the use-case and what you are doing here. (e.g. did you already have istio installed and now you want to install kiali separately? what helm command are you using to do this? that kind of thing)

Reply:
@jmazzitelli Also in order to avoid conflicts with names with the kiali chart embedded on istio super chart, we've uploaded in our internal chartmuseum a helm chart with a different name (kiali-istio) and uploaded the chart version we want to use. Then in our main chart we're using to deploy istio, we've setup the following in our `requirements.yaml`:

```
➜ cat requirements.yaml
dependencies:
  - name: istio
    version: 1.0.5
    repository: "@softonic"
  - name: kiali-istio
    version: 0.1.2
    repository: gs://kubertonic-charts/coreops
```

And in our values.yaml we've something like this:

```
kiali:
  enabled: false

kiali-istio:
  kiali:
    enabled: true
    contextPath: /
    ingress:
      enabled: false
    dashboard:
      grafanaURL: http://grafana.monitoring-v2.svc
      jaegerURL: http://jaeger-query.istio-system.svc:16686
    prometheusAddr: http://monitoring-v2-prometheus-server.monitoring-v2.svc
```

Let me know if you need more information!

Reply:
Last commit resolves a corner case where the nodeaffinity template is partially loaded and throws an error when trying to install kiali as an independent chart. Now it works supporting both installation methods, as a subchart and as an independent one. Do you need anything else from my side to merge this changes?